Source/ee/Vpu.h

```diff
@@ -39,7 +40,17 @@ class CVpu
 							CVpu(unsigned int, const VPUINIT&, CGIF&, uint8*, uint8*);
 	virtual					~CVpu();
 
-	void					Execute(bool);
+	void* operator new(size_t allocSize)
+	{
+		return framework_aligned_alloc(allocSize, 0x10);
```

Comment:
I'm getting the same build log with and without aligned allocation. There are other places where the compiler says objects might not be 16-byte aligned, but CVpu doesn't look like one of them. I'll remove the aligned allocation on my side.